### PR TITLE
[RESTEASY-2171] JsonBindingProvider use incorrect charset encoding

### PR DESCRIPTION
**Pull request by iweiss**
Issue: https://issues.jboss.org/browse/RESTEASY-2171

### CODE REVIEW

File: testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/ContainerConstants.java [hunk 1]
```diff
@@ -1,34 +1,38 @@
-package org.jboss.resteasy.test;
```
Comment (marekkopecky) [inline testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/ContainerConstants.java]: Wrong line endings? https://issues.jboss.org/browse/RESTEASY-2115

**Comment by marekkopecky** [comment]
https://github.com/resteasy/Resteasy/pull/1910#discussion_r264160330

This my comment was not fixed

File: testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/ContainerConstants.java (hunk 1 above)
Comment (iweiss) [reply testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/ContainerConstants.java]: This is due to how the merge was done, I think.

**Comment by iweiss** [comment]
The merge is wrong. Let me update this PR.

**Comment by marekkopecky** [comment]
Line encoding is not fixed after force push ...

**Comment by asoldano** [comment]
Honestly, I'm not holding this because of CRLFs; is there still any problem besides that @marekkopecky ? Otherwise I'll merge this. Thanks!

**Comment by marekkopecky** [comment]
@asoldano I don't see any functional issue with this PR, original issue seems to be fixed. Anyway, test for `@Produces("application/json;charset=us-ascii")` issue is missing.

**Comment by iweiss** [comment]
@marekkopecky `@Produces("application/json;charset=us-ascii")` added. How does it look?

**Comment by asoldano** [comment]
Thanks, merging

**Pull request merged**